testing/python/kernel/test_tilelang_kernel_int8_gemm_tcgen5.py

```
@@ -0,0 +1,83 @@
+
+    assert out_dtype in [T.int32], "Currently only int32 is supported"
+    assert accum_dtype in [T.int32], "Currently only int32 is supported"
+
+    if in_dtype is T.int8:
+        A = torch.randint(-128, 128, (M, K), device="cuda", dtype=torch.int8)
+        B = torch.randint(-128, 128, (N, K), device="cuda", dtype=torch.int8)
+    elif in_dtype is T.uint8:
+        A = torch.randint(0, 255, (M, K), device="cuda", dtype=torch.uint8)
+        B = torch.randint(0, 255, (N, K), device="cuda", dtype=torch.uint8)
+    else:
+        raise ValueError(f"Unsupported input dtype: {in_dtype}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)